cachi2/core/models/input.py

```diff
@@ -99,7 +99,7 @@ def _requirements_file_path_is_relative(cls, paths: list[Path]) -> list[Path]:
 
 
```

Comment:
Feels like this should be the first commit - before doing any functional changes to the code base start by trivial fixes, but that's just my terrible bias, feel free to happily ignore this comment :) .